UNRELEASED.md

```diff
@@ -1 +1,4 @@
-# Unreleased
\ No newline at end of file
+# Unreleased
+
+- react-native-bpk-component-select
+  - `valid` selects now align to behaviour on text inputs with regards to border colour.
```

Comment:
I think this can be clearer to users about what we've changed:

```suggestion
  - `BpkSelect` with `valid={true}` no longer has a green border to match the behaviour of `BpkTextInput` which also doesn't have one. 
```